S3/Config.py

```diff
@@ -73,6 +73,8 @@
     # In python 3, unicode -> str, and str -> bytes
     unicode = str
 
+PY32 = (sys.version_info >= (3, 2))
```

Comment:
PY3 will be enough (and consistent with what we use everywhere else) :-) 
(We are probably not compatible with python 3.0 to 3.4 that were not really a serious thing anyway)

Reply:
ty, pls squash since i made a silly commit message 👍